WordPress/src/main/java/org/wordpress/android/ui/JetpackConnectionWebViewActivity.java

```diff
@@ -77,12 +84,31 @@ private static void openJetpackConnectionFlow(Context context, String url, SiteM
         if (site != null) {
             intent.putExtra(WordPress.SITE, site);
         }
+        intent.putExtra(Source.KEY, source);
         context.startActivity(intent);
+        trackJetpackConnectionFlowStart(site, source);
+    }
+
+    static void trackWithSource(AnalyticsTracker.Stat stat, Source source) {
```

Comment:
I think this belongs to a util class or a common class if there is any that makes sense. Having it in an activity and then having to call `JetpackConnectionWebViewActivity.trackWithSource` from a different place feels wrong to me.

How would you feel about adding a new utils class that handles all these jetpack flow tracking?

Reply:
Sounds good!